### PR TITLE
added screen recorder

### PR DESCRIPTION
**Pull request by Phoenix-031**
Feature no #39 resolved.

### CODE REVIEW

File: PA.py [hunk 1]
```diff
@@ -68,6 +71,40 @@ def get_command():
         return query
 
 
+def screen_record():                                                                   #Screen Recording function
```
Comment (Umesh-01) [inline PA.py:74]: add this function in a separate file

Comment (Phoenix-031) [reply PA.py]: @Umesh-01 i have moved the function to a separate file please do check and see if it's okay :)...

File: PA.py [hunk 2]
```diff
@@ -68,8 +66,7 @@ def get_command():
         return query
 
 
-if __name__ == '__main__':
-
+if __name__ == '_main_':
```
Comment (Umesh-01) [inline PA.py:75]:
```suggestion
if __name__ == '__main__':
```

**Review by Umesh-01 — CHANGES_REQUESTED** [review]

**Review by Phoenix-031 — COMMENTED** [review]
made the necessary changes

**Review by Umesh-01 — APPROVED** [review]
LGTM 👍

**Pull request merged**